packages/framework/fluid-static/src/types.ts

```diff
@@ -57,6 +62,7 @@ export type SharedObjectClass<T extends IFluidLoadable> = {
  * @typeParam T - The class of the loadable object.
  * @public
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type LoadableObjectCtor<T extends IFluidLoadable> = new (...args: any[]) => T;
```

Comment:
you can clean up this and the other any's by doing:
```suggestion
export type LoadableObjectCtor<T extends IFluidLoadable=IFluidLoadable> = new (...args: unknown[]) => T;
```

Reply:
i'm wrong. the whole type hierarchy depends on any, and consumers need to cast 🤮

Reply:
Yeah. I briefly tried cleaning some of this up, but determined it was too big a project for this ☹️